docs/data/base/components/portal/portal.md

```diff
@@ -8,18 +8,28 @@ packageName: '@mui/base'
 
 # Portal
 
-<p class="description">The portal component renders its children into a new "subtree" outside of the current DOM hierarchy.</p>
+<p class="description">The <code>Portal</code> component lets you render its children into a DOM node that exists outside of its own DOM hierarchy.</p>
 
-The children of the portal component will be appended to the `container` specified.
-The component is used internally by the [`Modal`](/material-ui/react-modal/) and [`Popper`](/material-ui/react-popper/) components.
+`Portal` is a utility component built around [React's `createPortal()` API](https://reactjs.org/docs/portals.html).
+It gives you the functionality of `createPortal()` in a convenient component form.
 
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+:::info
+💡 According to [the React docs](https://reactjs.org/docs/portals.html), portals are useful when "you need the child element to visually 'break out' of its container."
+:::
 
-## Example
+The `Portal` component is used internally by Material UI's [`Modal`](material-ui/react-modal/) and [`Popper`](material-ui/react-popper/) components.
 
-{{"demo": "SimplePortal.js"}}
+## Basic usage
+
+Normally, children of a component are rendered within that component's DOM tree.
+But sometimes it's necessary to mount a child to a different location in the DOM.
```

Comment:
Why? We should give here example/explanation

Reply:
I did a little bit of rearranging to elaborate on this. Let me know what you think!